lorem.html

```diff
@@ -46,16 +46,17 @@
     />
     <meta name="theme-color" content="#ffffff" />
     <style>
-      img[id="i1"] {
+      img#i1 {
```

Comment:
What you have works, but if you're selecting a specific id, there's a special syntax for that!